tests/post/features/volume.feature

```diff
@@ -269,3 +269,36 @@ Feature: Volume management
         Then the Volume 'test-volume12-lvmlv' is 'Available'
         And the PersistentVolume 'test-volume12-lvmlv' has size '10Gi'
         And the device '/dev/test-vg-12/test-volume12-lvmlv' exists
+
+    Scenario: Test volume re-creation (lvmLogicalVolume force-lvcreate)
```

Comment:
From the CI build it seems you got a failure on this test in 3-nodes (flaky ?, weird, I don't see how this test could be flaky and the one for the single node succeeded :confused: )

Reply:
Yeah I need to debug this, haven't had time yet

Reply:
For reference: this was caused by an older version of LVM in CentOS 7, which doesn't prevent LV creation even if signatures were detected and not wiped. Added a note about this in the CHANGELOG, and skipped the test on CentOS 7 / RHEL 7 (actually, just testing the `osrelease` grain, but we only support RedHat-family OSes, so should be enough).